lib/Terminal/Print.pm6

```diff
@@ -198,22 +223,22 @@ class Terminal::Print {
     }
 
     multi method grid( Str $name ) {
-        die "No grid has been named $name" unless my $grid-index = %!grid-name-map{$name};
-        @!grids[$grid-index].grid;
+        die "No grid has been named $name" unless %!grid-name-map{$name}:exists;
+        @!grids[ %!grid-name-map{$name} ].grid;
```

Comment:
I'm not sure I see the purpose of this patch block, except to do the hash lookup twice.

Reply:
Because grid index can be 0 which is _False_ but valid value. `:exists` is cheap enough and the lookup itself is quick.

It could be `without my $grid-index = %!grid-name-map{$name}` which is more correct. But still, an existing key with undefined value is a bug.